packages/console/src/components/ItemPreview/index.module.scss

```diff
@@ -0,0 +1,22 @@
+@use '@/scss/underscore' as _;
+
+.item {
+  display: flex;
+  align-items: center;
+  white-space: nowrap;
+
+  > div:not(:first-child) {
+    margin-left: _.unit(4);
+  }
```

Comment:
There is only 1 child div inside `.item`, so what's this for?

Reply:
there are 2 actually

Reply:
this will provide more flexibility if we have more than 2 children